generators/php/model/src/generateTraits.ts

```diff
@@ -0,0 +1,34 @@
+import { ModelGeneratorContext } from "./ModelGeneratorContext";
+import { TraitGenerator } from "./trait/TraitGenerator";
+
+export function generateTraits(context: ModelGeneratorContext): void {
+    const extendedTypeIdsFromTypes: Set<string> = new Set(
```

Comment:
Easy lint fix here 
```suggestion
    const extendedTypeIdsFromTypes = new Set<string>(
```

Reply:
ty! looking at this prompted to make a fix and wait to create the set around the concatenated final list of typeids 👍